nats.go

```diff
@@ -839,14 +834,6 @@ func Timeout(t time.Duration) Option {
 	}
 }
 
-// JetStreamTimeout is an Option to set the timeout for access to the JetStream API
-func JetStreamTimeout(t time.Duration) Option {
```

Comment:
I use this one, what is it replaced with?

Reply:
Right now to be able to update the default JS API timeout, it has to be specified when creating the context, not when creating the connection for example:

```go
nc.JetStream(nats.MaxWait(5*time.Second))
```

Reply:
That is the way I use it.. So that is good.